### ISSUE TITLE
Model does not exist or cannot be loaded

### ISSUE THREAD

**Issue by XarlesSta** [juglab/n2v#87]
I managed to do the training of the data. Nevertheless, every time I tried to do the prediction, I have the same error(see below). Suggestions?

Note: I am trying this on my local PC using Fiji. I have not tried yep using Jupyter notebook. I did the training as well in Fiji. 

name=Thu Jun 25 16:43:12 PDT 2020 last checkpoint, description=null, cite=[{text=Krull, A. and Buchholz, T. and Jug, F. Noise2void - learning denoising from single noisy images.
Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition (2019), doi=arXiv:1811.10980}], authors=null, documentation=null, tags=[denoising, unet2d], license=null, format_version=0.2.0-csbdeep, language=java, framework=tensorflow, source=n2v, test_input=testinput.tif, test_output=testoutput.tif, inputs=[{name=input, axes=byxc, data_type=float32, data_range=[-inf, inf], halo=[0, 22, 22, 0], shape={min=[1, 16, 16, 1], step=[1, 16, 16, 0]}}], outputs=[{name=activation_11/Identity, axes=byxc, data_type=float32, data_range=[-inf, inf], shape={reference_input=input, scale=[1.0, 1.0, 1.0, 1.0], offset=[0, 0, 0, 0]}}], training={source=de.csbdresden.n2v.train.N2VTraining, kwargs={batchSize=64, learningRate=4.0E-4, trainDimensions=2, neighborhoodRadius=5, numEpochs=300, numStepsPerEpoch=200, patchShape=64, stepsFinished=60000}}, prediction={weights={source=./variables/variables}, preprocess=[{spec=de.csbdresden.n2v.predict.N2VPrediction::preprocess, kwargs={mean=[193.19666], stdDev=[30.815493]}}], postprocess=[{spec=de.csbdresden.n2v.predict.N2VPrediction::postprocess, kwargs={mean=[193.19666], stdDev=[30.815493]}}], dependencies=./dependencies.yaml}}
N2V prediction mean  : 193.19666
N2V prediction stdDev: 30.815493
[INFO] Using native TensorFlow version: TF 1.14.0 GPU (CUDA 10.0, CuDNN >= 7.4.1)
[INFO] Loading TensorFlow model Thu Jun 25 16:43:12 PDT 2020 last checkpoint 2020-06-25T23:43:12.563207Z from source file file:/C:/Users/GarzonCoral/Desktop/n2v-3850187345415030802.bioimage.io.zip
[INFO] Unpacking dependencies.yaml
java.io.FileNotFoundException: C:\Fiji.app\models\Thu Jun 25 16:43:12 PDT 2020 last checkpoint 2020-06-25T23:43:12.563207Z\dependencies.yaml (The filename, directory name, or volume label syntax is incorrect)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at net.imagej.tensorflow.util.UnpackUtil.unZip(UnpackUtil.java:149)
	at net.imagej.tensorflow.util.UnpackUtil.unZip(UnpackUtil.java:131)
	at net.imagej.tensorflow.DefaultTensorFlowService.downloadAndUnpackResource(DefaultTensorFlowService.java:407)
	at net.imagej.tensorflow.DefaultTensorFlowService.modelDir(DefaultTensorFlowService.java:384)
	at net.imagej.tensorflow.DefaultTensorFlowService.loadCachedModel(DefaultTensorFlowService.java:131)
	at net.imagej.modelzoo.consumer.model.tensorflow.TensorFlowModel.loadModelFile(TensorFlowModel.java:145)
	at net.imagej.modelzoo.consumer.model.tensorflow.TensorFlowModel.loadModel(TensorFlowModel.java:110)
	at net.imagej.modelzoo.DefaultModelZooArchive.createModelInstance(DefaultModelZooArchive.java:101)
	at net.imagej.modelzoo.consumer.DefaultModelZooPrediction.loadModel(DefaultModelZooPrediction.java:107)
	at net.imagej.modelzoo.consumer.DefaultModelZooPrediction.run(DefaultModelZooPrediction.java:81)
	at de.csbdresden.n2v.predict.N2VPrediction.run(N2VPrediction.java:89)
	at de.csbdresden.n2v.predict.N2VPrediction.predictPadded(N2VPrediction.java:105)
	at de.csbdresden.n2v.command.N2VPredictCommand.run(N2VPredictCommand.java:123)
	at org.scijava.command.CommandModule.run(CommandModule.java:199)
	at org.scijava.module.ModuleRunner.run(ModuleRunner.java:168)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:127)
	at org.scijava.module.ModuleRunner.call(ModuleRunner.java:66)
	at org.scijava.thread.DefaultThreadService.lambda$wrap$2(DefaultThreadService.java:228)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[ERROR] Model does not exist or cannot be loaded. Exiting.

**Comment by tibuch** [comment]
Hi @XarlesSta 

Thank you for reporting. I will move this issue to N2V in Fiji.
